source/_layouts/has_both_sidebars.liquid

```diff
@@ -0,0 +1,30 @@
+---
+layout: base
+---
+
+<div class="sl-l-medium-holy-grail">
+  <div class="sl-l-medium-holy-grail__body">
+    {% render 'navigation', contents: navigation %}
+
+    <div class="sl-l-medium-holy-grail__main">
+      <div class="sl-l-medium-holy-grail">
+        <div class="sl-l-medium-holy-grail__body">
+          {% if complementary_content %}
+            <div class="sl-l-large-holy-grail__complementary{% if table_of_contents %} sl-l-large-holy-grail__complementary--contents{% endif %}">
+              {{ complementary_content | markdown }}
+            </div>
+          {% endif %}
+
+          <div class="sl-l-large-holy-grail__main docSearch-content">
+            {% assign is_typedoc = page | isTypedoc %}
+            <div {% unless no_container or is_typedoc %}class="sl-l-container sl-l-container--small"{% endunless %}>
```

Comment:
In the old layout, this div also was given `.sl-l-container--overview` if the page had `intro: true`. But there's no record of any page using `intro` anywhere, so I removed it. I think it was likely outdated.

Reply:
Yeah, my guess is it was a typo initially, or at some point -- but I think it's old enough that the styles aren't wanted anymore? At least when I tried applying them it looked not good.